src/jwt-store.ts

```diff
@@ -208,7 +208,11 @@ const expiredP = (roomName: string, jwt: string): boolean => {
 
   try {
     const payload = jwt_decode(jwt);
-    return payload.exp < now;
+    const isWeb3 =
+      payload.context &&
+      payload.context["x-brave-features"] &&
+      payload.context["x-brave-features"].web3 === "true";
+    return payload.exp < now || isWeb3;
```

Comment:
if normal or refresh JWT is a web3 one, mark it as "expired" so it will be removed during confab garbage collection